### DISCUSSION TITLE
with #168

### DISCUSSION THREAD

**Discussion by 0xameer** [gvolpe/nix-config#184] [Q&A] [answered]
@gvolpe  #168 
a) how does drone + gitlab handle -not build for this - trigger?
b) my binary cache\ substitutor  for rest of the nar.xz i need for nix , work, but i still get http error for garnix ,despite
`extraOptions = ''
        experimental-features = nix-command flakes
        binary-caches-parallel-connections = 3
        connect-timeout = 5
        '';
  binaryCaches          = [ "https://cache.iog.io"
                            "https://aseipp-nix-cache.global.ssl.fastly.net"
                          ];
`
sets to reproduce
`./build fresh-install --keep-going --fallback # requires sudo`
error: unable to download 'https://cache.garnix.io/nar/xawrm4qil5k6yvmxqdkqczlqsv865l4g-0limmb6d06352pqsdf0nspplzpclvlp0p5l023acs2izz4f2rm6i.nar': HTTP error 200 ('') (curl error: Stream error in the HTTP/2 framing layer)
error: some substitutes for the outputs of derivation '/nix/store/c3d2m2j0dys10191k7s5yb6r92rf23m8-fontconfig-conf.drv' failed (usually happens due to networking issues); try '--fallback' to build derivation from source
error: 1 dependencies of derivation '/nix/store/kp3kw2a3105xjlsj6kjmmm1g17mxsbwg-fontconfig-etc.drv' failed to build
postscript
I have modded build script for my optiplex
Namely , cp-ed my nix generated hardware configuration to system/machines/optiplex , in my fork 
however
nix build github:carnotweat/nix-config#homeConfigurations.gvolpe-hdmi.activationPackage
works on my system

**Comment by gvolpe** [comment]
Hard to tell without seeing your configuration files.

**Reply by gvolpe** [reply, marked as answer]
Ok I see you're trying to build my configuration. The substituters are defined [here](https://github.com/carnotweat/nix-config/blob/master/system/cachix/gvolpe-nixos.nix) for the `root` user, but for your user you need to [define them](https://github.com/carnotweat/nix-config/blob/master/home/daemon.conf) under `~/.config/nix/nix.conf`.

Also bear in mind that my home configuration has a lot of things that are tailored to my needs and might not necessarily work for you.
